### PR TITLE
feature : added input component

### PR DESCRIPTION
**Pull request by aliamaanansari**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+        <!-- will be handled by js -->
+      </div>
+      <div class="mid-div">
+        <hr />
+
+        <section class="parent-section">
+          <h1>Avatar :-</h1>
+          <div class="avatar-container">
+            <div class="avatar box-xxl">
+              <img src="/assets/images/user1.jpg" alt="" />
```
Comment (krishanu2911) [inline index.html]: fill the alt tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+            </div>
+            <div class="avatar box-xxl">K T</div>
+            <div class="avatar box-xxl">
+              <img src="/assets/images/user2.webp" alt="" />
```
Comment (krishanu2911) [inline index.html]: same here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+            </div>
+          </div>
+          <p>Avatar: The image that represents the user.</p>
+          <p>The Avatar component comes in different sizes -</p>
+          <h3>Large -</h3>
+          <div id="large-Avatar" class="avatar-container">
+            <div class="avatar box-xxl">
+              <img src="/assets/images/user1.jpg" alt="" />
```
Comment (krishanu2911) [inline index.html]: same here

File: index.html [hunk 4]
```diff
@@ -0,0 +1,677 @@
...
+            </div>
+            <div class="avatar box-xxl">K T</div>
+            <div class="avatar box-xxl">
+              <img src="/assets/images/user2.webp" alt="" />
```
Comment (krishanu2911) [inline index.html]: here too

**Review by krishanu2911 — APPROVED** [review]
No changes required good to go , Just add a suggestion to not leave the alt tag empty

File: home.css [hunk 5]
```diff
@@ -0,0 +1,638 @@
...
+  background-color: white;
+  width: 20%;
+  height: calc(100vh - 5rem);
+}
+
+.nav-item {
+  margin: 0.5rem 1rem;
+  font-weight: 500;
+  color: #2d3748;
+  font-size: 14px;
```
Comment (aatishpandey) [inline home.css]: you can use rem here

File: home.css [hunk 6]
```diff
@@ -0,0 +1,638 @@
...
+/* Avatar */
+.avatar-container {
+  margin: 1rem 0;
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+.avatar {
+  background-color: grey;
+  width: 50px;
```
Comment (aatishpandey) [inline home.css]: same here

**Review by aatishpandey — COMMENTED** [review]
Looks good. Just a suggestion to use rem instead of px for responsiveness. There are multiple places you have used px,   any specific use case?

**Pull request merged**